tornado_handlers/error_labels.py

```diff
@@ -0,0 +1,54 @@
+"""
+Tornado handler for updating the error label information in the database
+"""
+from __future__ import print_function
+import sys
+import os
+from html import escape
+import sqlite3
+import tornado.web
+
+# this is needed for the following imports
+sys.path.append(os.path.join(os.path.dirname(os.path.realpath(__file__)), 'plot_app'))
+from config import *
+from db_entry import *
+from helper import validate_log_id, validate_error_labels_and_get_ids
+
+class UpdateErrorLabelHandler(tornado.web.RequestHandler):
+    """ Update the error label of a flight log."""
+
+    def post(self, *args, **kwargs):
+
+        log_id = escape(self.get_argument('log'))
+        if not validate_log_id(log_id):
+            raise tornado.web.HTTPError(400, 'Invalid Parameter')
+
+        text_error_labels = escape(self.get_argument('labels')).split(',')[:-1]
```

Comment:
Can you make sure the sender does not add the `,` at the end, so that `[:-1]` is not needed?

Reply:
the point here is that this also works nicely when the empty string is send as it returns an empty list and thus saves me a few ifs and else's. But I can change it, if you prefer a solution that might be easier to follow.

Reply:
Right, `split` has this strange semantics. Since this is a web API (even though only used internally), I somewhat prefer it to be simple & intuitive to use. In this case, if the last comma is not added, the last element is dropped for non-obvious reasons.
What about using JSON?